src/platform/icelake/include/platform/lib/memory.h

```diff
@@ -183,7 +183,10 @@
 #define SRAM_REG_FW_TRACEP			0x8
 #define SRAM_REG_FW_IPC_RECEIVED_COUNT		0xc
 #define SRAM_REG_FW_IPC_PROCESSED_COUNT		0x10
-#define SRAM_REG_FW_END				0x14
+#define SRAM_REG_FW_TRACEP_CORE1		0x14
+#define SRAM_REG_FW_TRACEP_CORE2		0x18
+#define SRAM_REG_FW_TRACEP_CORE3		0x1c
```

Comment:
I'd not go for enumerating TRACEP for all cores, I think it's better to just have base for slave core and compute others with macros. Especially for ICL, where for JSL 2&3 will not be valid.